web/components/home/Questions.js

```diff
@@ -27,11 +27,22 @@ export const QUESTIONS = Object.freeze({
       ),
     },
     {
-      'How can I officially sign up?': (
-        <p className="margin-top-none">Registration on our website will open in November.&nbsp;
-          <a href="http://eepurl.com/dF3b7n">
-            Let us know your email and we&apos;ll tell you once it does!
-          </a>
+      'Application deadline?': (
```

Comment:
Since the rest of the FAQ questions are full sentences, can we do the same here? (`When is the application deadline?`)